iphone/Classes/TiContactsPerson.h

```diff
@@ -31,19 +37,24 @@
 
 -(id)_initWithPageContext:(id<TiEvaluator>)context recordId:(ABRecordID)id_ module:(ContactsModule*)module_;
 -(id)_initWithPageContext:(id<TiEvaluator>)context person:(ABRecordRef)person_ module:(ContactsModule*)module_;
+
 @property(readonly,nonatomic) NSString* identifier;
+@property(assign, nonatomic) id<TiContactsPersonUpdateObserver> observer;
+
 +(NSDictionary*)iOS9multiValueLabels;
 +(NSDictionary*)iOS9propertyKeys;
--(id)_initWithPageContext:(id<TiEvaluator>)context contactId:(CNMutableContact*)person_ module:(ContactsModule*)module_;
+-(id)_initWithPageContext:(id<TiEvaluator>)context
+                contactId:(CNMutableContact*)person_
+                   module:(ContactsModule*)module_
+                 observer:(id<TiContactsPersonUpdateObserver>) observer_;
 -(CNSaveRequest*)getSaveRequestForDeletion;
 -(CNSaveRequest*)getSaveRequestForAddition:(NSString*)containerIdentifier;
 -(CNSaveRequest*)getSaveRequestForAddToGroup: (CNMutableGroup*) group;
 -(CNSaveRequest*)getSaveRequestForRemoveFromGroup: (CNMutableGroup*) group;
 -(void)updateiOS9ContactProperties;
 -(CNMutableContact*)nativePerson;
+#endif
 -(id)valueForUndefinedKey:(NSString *)key;
 -(NSString*)fullName;
```

Comment:
Is there a reason why we should access the fullName when {{Ti.Contacts.Person}} is not defined?